test cases/common/93 selfbuilt custom/meson.build

```diff
@@ -14,3 +14,11 @@ hfile = custom_target('datah',
 main = executable('mainprog', 'mainprog.cpp', hfile)
 
 test('maintest', main)
+
+lib = library('libtool', 'tool.cpp')
+
+ctlib = custom_target('ctlib',
+  output : 'ctlib.out',
+  capture : true,
+  command : [main, lib],
```

Comment:
Custom targets are not built by default, thus this is not actually run. And it would probably not run on Windows because shared libs are nor directly runnable (IIRC, and they might not be runnable on Linux either but I don't know the specifics).

A better test would use the generated file as an argument to some other command (such as a script that checks that the output file is actually generated).

Reply:
yes, the point was rather to check that meson accept the argument, not really to run it. I have updated it so it will run by default with a basic test (not perfect though, I don't think we need to do much more).

Reply:
@jpakkane It's exceedingly rare, but Linux shared libraries can be given entry points and then be executed. In particular, GNU libc `/lib64/libc.so.6` and the dynamic loader `/lib64/ld-linux-x86-64.so.2`, executed with no arguments, print information about themselves and exit, and obviously `/lib64/ld-linux-x86-64.so.2` serves as the "interpreter" of all dynamically-linked programs on x86-64 Linux.